fudge-ratio/src/main/webapp/calculator/main.js

```diff
@@ -1,83 +1,16 @@
-import { saveSessionState, recoverSessionState } from "../code/save.js";
```

Comment:
This commit is where the real work is, now, each file, have the responsibility of painting to the screen to, but thanks to that we can have many more globals, so everything is more modular